ProiectHangman/src/App.tsx

```diff
@@ -0,0 +1,104 @@
+import { useCallback, useEffect, useState } from "react";
+import { HangmanDrawing } from "./HangmanDrawing";
+import { HangmanWord } from "./HangmanWord";
+import { Keyboard } from "./Keyboard";
+import words from "./wordList.json";
+
+function getRandomWordObject() {
+  const randomIndex = Math.floor(Math.random() * words.length);
+  return words[randomIndex];
+}
+
+function App() {
+  const [wordObject, setWordObject] = useState(getRandomWordObject);
+  const [guessedLetters, setGuessedLetters] = useState<string[]>([]);
+  const { word: wordToGuess, hint } = wordObject;
+  const incorrectLetters = guessedLetters.filter(
+    letter => !wordToGuess.includes(letter)
+  );
+
+  const isLoser = incorrectLetters.length >= 6;
+  const isWinner = wordToGuess
+    .split("")
+    .every(letter => guessedLetters.includes(letter));
+
+  const addGuessedLetter = useCallback(
+    (letter: string) => {
+      if (guessedLetters.includes(letter) || isLoser || isWinner) return;
+
+      setGuessedLetters(currentLetters => [...currentLetters, letter]);
+    },
+    [guessedLetters, isWinner, isLoser]
+  );
+
+  useEffect(() => {
+    const handleKeyPress = (e: KeyboardEvent) => {
+      const key = e.key.toLowerCase();
+      if (!key.match(/^[a-z]$/)) return;
+
+      e.preventDefault();
+      addGuessedLetter(key);
+    };
+
+    document.addEventListener("keypress", handleKeyPress);
+
+    return () => {
+      document.removeEventListener("keypress", handleKeyPress);
+    };
+  }, [guessedLetters]);
+
+  useEffect(() => {
+    const handleEnterPress = (e: KeyboardEvent) => {
+      if (e.key !== "Enter") return;
+
+      e.preventDefault();
+      setGuessedLetters([]);
+      setWordObject(getRandomWordObject());
+    };
+
+    document.addEventListener("keypress", handleEnterPress);
+
+    return () => {
+      document.removeEventListener("keypress", handleEnterPress);
+    };
+  }, []);
+
+  return (
+    <div
+      style={{
+        maxWidth: "800px",
+        display: "flex",
+        flexDirection: "column",
+        gap: "2rem",
+        margin: "0 auto",
+        alignItems: "center",
+      }}
```

Comment:
De ce nu ai pus astea ca și clasă? 

Dacă îți place să lucrezi așa, mai bine folosești o librărie de CSS-în-JS.
Vezi [lista asta](https://nextjs.org/docs/app/building-your-application/styling/css-in-js) de la documentația Next. Toate ar trebui să poată fi folosite și fără Next. Aș recomanda [Emotion](https://emotion.sh/docs/@emotion/react)